config/routes.rb

```diff
@@ -3,12 +3,18 @@
 Rails.application.routes.draw do
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
+
   devise_for :users, controllers: { session: 'users/session' }
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'tasklists#index'
   resources :tasklists do
-    resources :tasks
+    member do
+      get 'render_tasks'
+    end
+    resources :tasks do
+      member do
+        patch 'checkbox_update'
```

Comment:
`checkbox_update` needs a better name. I don't understand anything about the application with such a generic name

Reply:
resolved